actors/runtime/indices/indices.go

```diff
@@ -181,22 +169,23 @@ func (inds *IndicesImpl) StoragePower(
 	
 	// minerNominalPower * minerPledgeCollateral / requiredPledge
 	// this is likely to change
+	// PARAM_FINISH()
 	return big.Div(big.Mul(minerNominalPower, minerPledgeCollateral), requiredPledge)
 }
 
 func (inds *IndicesImpl) StoragePowerProportion(
 	minerStoragePower abi.StoragePower,
 ) big.Int {
-	PARAM_FINISH()
-	panic("")
+	// PARAM_FINISH()
+	return big.NewInt(0)
 }
 
 func (inds *IndicesImpl) CurrEpochBlockReward() abi.TokenAmount {
 	// total block reward allocated for CurrEpoch
 	// each expected winner get an equal share of this reward
 	// computed as a function of NetworkKPI, NetworkTarget, LastEpochReward, TotalUnmminedFIL, etc
-	PARAM_FINISH()
-	panic("")
+	// PARAM_FINISH()
+	return abi.NewTokenAmount(0)
```

Comment:
At least make this non-zero so a network could progress.

Also ⚠️ here is a function that does not have the right type, and fails to declare the parameters it needs.

Reply:
ack, per input type, we were assuming this is computed from the state tree before but I think we can update the input type here now.

Reply:
will leave out the details for a future PR